### DISCUSSION TITLE
Unexpected loss of document on running out of disk space

### DISCUSSION THREAD

**Discussion by datavectors** [Zettlr/Zettlr#3730] [General]
I was editing an important document open in Zettlr. Foolishly I did not have a backup.
I was using a separate application which used in error caused me to run out of disk space. 

I have now released some disk space and the PC has not been shut down yet .

But the file I had open in Zettlr has lost all content. The word count is zero.

Some Zettlr messages did pop up when I ran out of disk space but I do not recollect them. I was too busy trying to recover space.

This identifies a flaw I will need to correct to ensure that all files are regularly backed up when in use, perhaps every five minutes.
But regarding my plight is there any way of recovering this one file which was open at the time? Other Zettlr files (unopened) were spared the axe.

**Comment by nathanlesage** [comment]
> But regarding my plight is there any way of recovering this one file which was open at the time?

I'm afraid no. That's not how computers work, unfortunately.
